bumping_dependencies.md

```diff
@@ -1,6 +1,52 @@
-# Bumping dependencies
+# Bumping openstack-k8s-operator dependencies
 
-If you have backward incompatible changes in any of the operators `/api`
+The dependencies for all openstack-k8s-operators currently use [psuedo-versions](https://go.dev/ref/mod#pseudo-versions). This includes the dependencies for lib-common and all operator* repositories under the openstack-k8s-operators github org.
+
+## bumping all dependencies in a local operator repo
+
+Each operator has a Makefile target called 'make force-bump'. Running this target will bump the dependencies
+for all openstack-k8s-operator dependencies in one step. After running this step if there are struct changes
+you may need to re-run 'make manifests generate' to apply them. If you are using the openstack-operator
+then you would also need to run a 'make bindata' step last to update all the cached bindata files
+for the OpenStack initialization resource (see more on this below).
+
+If you prefer you can also manually update a single dependency using a command like:
+```
+go get github.com/openstack-k8s-operators/glance-operator/api@main
+```
+
+## github actions to automatically bump dependencies
+
+There are two github action workflows that can be used to automate bumping dependencies for each operator repository:
+
+- "Manually Trigger a Force Bump PR" action workflow can be executed on demand to create or update a pull request that bumps operator dependencies for the selected branch. NOTE: this action only runs if the workflow exists on that branch (main and FR3 going forward)
+
+- "Scheduled Force Bump PR" action workflow is executed automatically on weekends. The actions run across the repositories in stages, a couple of pull requests are generated each hour (so as not to overburden CI all at once). This workflow runs
+on both the 'main' and last 'feature release' branches.
```

Comment:
this line break is at a strange place. I suggest to wrap near 80 or 120 chars instead